README.md

```diff
@@ -6,3 +6,14 @@
 
 ## Structure
 ![RCLC Structure with bg](https://user-images.githubusercontent.com/35516367/184010638-25389f58-7de6-4d69-bc0b-5467f6618ef8.png)
+
+## Install
+The `~/.rclc/` dir needs to be created. So do the two FIFOs. Do this by running the following.
```

Comment:
This feels like a serial killer talking, but it's fine for now.

Reply:
> Two FIFOs and the `~/.rclc/` directory need to be created.